src/stories/TableWithInlineEditing.stories.js

```diff
@@ -70,19 +69,33 @@ storiesOf('TableWithInlineEditing', module)
             <div>{story()}</div>
         </ThemeProvider>
     ))
-    .add('allowed to edit', () => (
+    .add('allowed to edit, add and delete', () => (
+        <TableWithInlineEditing
+            columnsInfo={columnsInfo}
+            content={content}
+            updateContent={{}}
+            allowedToEdit
+            allowedToCreate
+            allowedToDelete
+        />
+    ))
+    .add('allowed to edit and add', () => (
         <TableWithInlineEditing
             columnsInfo={columnsInfo}
             content={content}
             updateContent={{}}
             allowedToEdit
+            allowedToCreate
         />
     ))
-    .add('without edit permission', () => (
+    .add('allowed to edit only', () => (
         <TableWithInlineEditing
             columnsInfo={columnsInfo}
             content={content}
             updateContent={{}}
-            allowedToEdit={false}
+            allowedToEdit
         />
+    ))
+    .add('Not allowed to edit', () => (
+        <TableWithInlineEditing columnsInfo={columnsInfo} content={content} updateContent={{}} />
```

Comment:
same again. Also you should pass 

`allowedToEdit={false}`

or perhaps make false the defaultProp to reduce keystrokes?